backend/asset/timedmetadata.images.go

```diff
@@ -0,0 +1,76 @@
+package asset
+
+import (
+	"context"
+	"os"
+	"path"
+	"sync"
+
+	"github.com/ansel1/merry/v2"
+	"github.com/bcc-code/bcc-media-platform/backend/files"
+	"github.com/bcc-code/mediabank-bridge/log"
+	"golang.org/x/sync/errgroup"
+)
+
+// ingestImagesFromS3 downloads images from S3 and uploads them to the platform
+func ingestImagesFromS3(ctx context.Context, imagePaths []string, services externalServices, config config) (map[string]string, error) {
+	s3client := services.GetS3Client()
+	tempDir, err := os.MkdirTemp(config.GetTempDir(), "timedmetadata")
```

Comment:
Is there a specific reason we are not using `os.TempDir()` but are instead introducing a whole new config var?